packages/addons/service/argononecontrol/package.mk

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2025-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="argononecontrol"
+PKG_VERSION="1.1.6"
+PKG_SHA256="9c15299d48efe5a138ce3ce8d0b78bb7d2f5317988e67346295629af752967ee"
+PKG_REV="0"
+PKG_ARCH="arm aarch64"
+PKG_MAINTAINER="HungerHa"
+PKG_LICENSE="MIT"
+PKG_SITE="https://github.com/HungerHa/libreelec_addon_argononecontrol"
+PKG_URL="https://github.com/HungerHa/libreelec_addon_argononecontrol/archive/refs/tags/v$PKG_VERSION.tar.gz"
+PKG_SECTION="service"
+PKG_SHORTDESC="Argon ONE Control"
+PKG_LONGDESC="Support for the RPi4/5 Argon ONE cases. Installs services to manage ArgonForty devices, such as power button, fan speed and Argon REMOTE. This also activates I2C, IR receiver and UART (requires a one-time restart)."
```

Comment:
Suggested change:

`PKG_LONGDESC="Support for RPi4/5 Argon ONE case features including the power button, fan speed, and the Argon IR remote. One-time restart required.`